setup.py

```diff
@@ -46,4 +47,12 @@ def read(fname):
         },
     test_suite='nose.collector',
     zip_safe=False,
+    data_files=[
+        ('socorro/external/postgresql/raw_sql/procs',
+            glob.glob('socorro/external/postgresql/raw_sql/procs/*.sql')),
+        ('socorro/external/postgresql/raw_sql/views',
+            glob.glob('socorro/external/postgresql/raw_sql/views/*.sql')),
+        ('socorro/external/postgresql/raw_sql/types',
+            glob.glob('socorro/external/postgresql/raw_sql/types/*.sql')),
+    ],
```

Comment:
For the record, all of this could be replaced with adding 
```
include socorro/external/postgresl/raw_sql/*/*.sql
```
in the `MANIFEST.in` file. 

The question is, what's the difference? Is one better than the other?